holoplot/__init__.py

```diff
@@ -0,0 +1,385 @@
+import holoviews
```

Comment:
I strongly prefer all .py files to have a module docstring that explains what this module is about, but I see that even holoviews doesn't have one for its `__init__.py`, so you must disagree! :-)

Reply:
I don't disagree, HoloViews is horribly underdocumented and I don't want to repeat that here.